.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,12 @@
-Fixes #
-
 <!--
-Please follow the naming conventions in the "guidelines for contributing" link above or the pull request may be rejected.
+Naming convention for Pull Requests:
 
-Also, do consider outlining the solution taken as doing so will likely help in the reviewing process
-(e.g., when the pull request involves non-trivial changes)
+<Issue title the pull request fixes><space>#<Issue number>
 -->
+
+Fixes #
+
+**What the Problem Was**
```

Comment:
This needs not be here as the issue being fixed should provide sufficient information.